NOTES_FOR_MAINTAINERS.md

```diff
@@ -27,11 +27,62 @@ The script output is designed to be deterministic; if the vega-lite version
 is not changed, then running the script should overwrite the schema wrappers
 with identical copies.
 
-## Updating the Vega-Lite version
+## Updating Vega versions
+All versions are maintained in [pyproject.toml](pyproject.toml).
 
-The vega & vega-lite versions for the Python code can be updated by manually
-changing the ``SCHEMA_VERSION`` definition within
-``tools/generate_schema_wrapper.py``, and then re-running the script.
+### Python Packages
+
+Projects which publish a package to PyPI are listed with a version bound in one of the following tables:
+
+- [`project.dependencies`](https://packaging.python.org/en/latest/specifications/pyproject-toml/#dependencies-optional-dependencies): Published dependencies.
+- [`project.optional-dependencies`](https://packaging.python.org/en/latest/specifications/pyproject-toml/#dependencies-optional-dependencies): Published optional dependencies, or "extras".
+- [`dependency-groups`](https://peps.python.org/pep-0735/): Local dependencies for development.
+
+> [!NOTE]
+> All are currently declared in sub-tables of `project.optional-dependencies`
+
+#### `vl-convert`
+
+We need to ensure that [vl-convert](https://github.com/vega/vl-convert) includes support for the new Vega-Lite version. 
+Check the [vl-convert releases](https://github.com/vega/vl-convert/releases) to find the minimum
+version of `vl-convert` that includes support for the desired version of Vega-Lite (and [open
+an issue](https://github.com/vega/vl-convert/issues) if this version hasn't been
+included in a released yet).
+
+#### `vegafusion`
+
+
+> [!WARNING]
+> *TODO: Find out how the constraint for `vegafusion` is decided*
```

Comment:
@jonmmease do you have any thoughts on what should go here? 

AFAIK we haven't documented this before - but it seems like something that would be helpful if you weren't available

Reply:
The VegaFusion constraint isn't tied to the Vega-Lite version (since VegaFusion works at the Vega level). In theory it could be connected to the Vega version, but Vega has been so stable that this hasn't come up in practice.

I think the VegaFusion version constraint is determined by the API we depend on, just like a regular Python dependency.

Reply:
> I think the VegaFusion version constraint is determined by the API we depend on, just like a regular Python dependency.

Thanks @jonmmease, I think I'll just remove that section then

Reply:
Resolved in (https://github.com/vega/altair/pull/3720/commits/ac689ddb4f5242fb7a0d3a645a0b493edb090705)